osgtest/tests/test_410_jobs.py

```diff
@@ -12,6 +12,13 @@
 
 class TestRunJobs(osgunittest.OSGTestCase):
 
+    def tearDown(self):
+        env_vars = ('_condor_SCITOKENS_FILE',
+                    'BEARER_TOKEN_FILE')
+
+        for env_var in env_vars:
+            os.environ.pop(env_var, None)
```

Comment:
I was going to make the same comment about `del d[k]`.  Looks great how you've done it though.

> The fewer tabs the better

well if that's the case...

```suggestion
        [ os.environ.pop(env_var, None) for env_var in env_vars ]
```

(whose returned list will also be thrown to the wind)